docs/triply-api/index.md

```diff
@@ -824,8 +824,233 @@ Result:
+```
+correspond to the below graphql types:
+```graphql
+type Book {
+  id:ID!
+  author:[Person]!
+  audio:[ExternalIri]!
+}
+
+type Person {
+  id:ID!
+  name:[XsdString]!
+}
+```
+#### IDs 
+IDs represent the IRI of each object. This ID is unique.
+#### Naming
+In order to name the GraphQL types in correspondence to shapes, we follow the below conventions:
+- For object types, we use the `sh:targetClass` of the node shape.
+- For object type fields, we use the `sh:path` of the property shape.
+
+More specifically, the name comes from the part of the IRI after the last `#` or otherwise the last `/`, converted from kebab-case to camelCase. 
+
+Notice that if the selected name is illegal or causes a name collision, we'll return an error informing the user about the problem and ignore this type or field. 
+
+ ##### Renaming
+ Shape designers are able use their custom names by using a special property: `<https://triplydb.com/Triply/sparql/graphqlName>`.
+ More specifically, the designer has to add a triple with :
+ - for object types, the class IRI 
+ - for fields, the IRI of the property shape
+
+as a subject, the above-mentioned predicate and a string literal with the custom name as object.
+
+ If we wanted to rename using the first example of the section, we would do:
+```turtle
+shp:Book a sh:NodeShape;
+         sh:targetClass sdo:Book;
+         sh:property [ 
+           sh:path dc:title;
+           sh:graphqlName "name" // Renaming the object type field
+           sh:datatype xsd:string.]
+sdo:Book triply:graphqlName "PieceOfArt". // Renaming the object type
+```
+Then the corresponding object type would be:
+```graphql
+type PieceOfArt {
+  id:ID!
+  name:[XsdString]!
+}
+```
+### Queries
+The user can query for objects using their unique ID. Also, they can query for objects of a specific type along with fields, and get nested information. Last, the user can get information by filtering results. Let's see some important concepts.
+
+
+#### Global Object identification
+For several reasons, the user should be able to query an object by their unique ID. This is possible using global object identification, using the `node(id:ID)` query. 
+An example:
+
+```graphql
+{
+  node(id: "https://example.com/Anna") {
+    id
+  }
+}
+```
+
+For more information on global object identification, see [graphql specification](https://graphql.org/learn/global-object-identification/). 
+
+#### Pagination
+A simple query would be:
+ ```graphql
+{
+  BookConnection {
+    edges {
+      node {
+        id
+        title
+      }
+    }
+  }
+}
+ ```
+The results would include the IRIs of books together with their titles and would be paginated.
+
+In order to allow the user to use limit and offset, our GraphQL implementation supports **cursor-based pagination using connections**. For more information, please visit the Relay project's [cursor-based connection pagination specification](https://relay.dev/graphql/connections.htm).
```

Comment:
Please don't mention `limit` and `offset` :scream: We don't support them.

```suggestion
In order to paginate through a large number of results, our GraphQL implementation supports **cursor-based pagination using connections**. For more information, please visit the Relay project's [cursor-based connection pagination specification](https://relay.dev/graphql/connections.htm).
```

Reply:
oops! But they can do first and after right? But I think the way you phrased it is very good already